### ISSUE TITLE
Add link of the GitHub Issue to the License Information page

### ISSUE THREAD

**Issue by BanulaKumarage** [spdx/spdx-online-tools#453]
Even though there is a link to access the request of the online tool from the Github issue, there is no link to access the GitHub issue from the online tool.

**Comment by BassCoder2808** [comment]
Yes, I think this would be helpful to the user as currently, we are only opening a new tab for discussion for the github issues as per [Redirect on Successful License Submission](https://github.com/spdx/spdx-online-tools/issues/449).

**Comment by BassCoder2808** [comment]
Actually, this issue can be closed as #452 intends to solve the same with some other minor changes.

**Comment by BanulaKumarage** [comment]
> Actually, this issue can be closed as #452 intends to solve the same with some other minor changes.

@BassCoder2808 It seems like there is no connection between those two issues. So I think #454 can fix this issue instead of solving it under #452

**Comment by jlovejoy** [comment]
add link to Github issue created from license submission to list of submitted licenses at https://tools.spdx.org/app/license_requests/

@BassCoder2808 - has this been solved for and merged?

**Comment by BassCoder2808** [comment]
Hi @jlovejoy actually there has been some miscommunications like there are 2 issues and 2 PRs that solve the 2 issues separately which in the end intend to do the same task.
So what I feel is that we can move forward with #452 (just because it was opened earlier and solves one more issue)
and the PR to move forward with #456

**Comment by jlovejoy** [comment]
@BassCoder2808 - I"m confused this issue is about adding the Github link to the license info page and #452  is about removing the Standard License Header field from the output in the issue, so I think we should leave both issues?

but is it that the PR #456 solves both of these issues?

**Comment by BassCoder2808** [comment]
Hi @jlovejoy yes actually #452 the issue which you had created also intended to do the same.
Yes, #456 intends to solve both the task which you have mentioned above regarding the "Standard License Header" as well as the "open issue on new tab"

**Comment by jlovejoy** [comment]
was this issue intended to add a link to the Github issue to individual license pages in the tool like https://tools.spdx.org/app/license_requests/251/ 
or to the list here https://tools.spdx.org/app/license_requests/ ?

I was thinking the latter, but in any case, it is not there, so I don't think this is resolved yet

**Comment by BassCoder2808** [comment]
As far as I remember @jlovejoy this was intended to open a new issues tab for that particular issue only because my thinking was that, the user will be focused on the currently submitted license only and not towards the general list
